samples/Demo Data/README.md

```diff
@@ -0,0 +1,21 @@
+# Demo Data Project
+
+The purpose of this project is to exercise E2E capabilities of the Datadog SDK, and produce
+demo-able data with some amount of variance.
+
+## Running
+
+The Datadog configuration is already setup, but is lacking the client token and application ID. These
+can be populated by running `./add-keys.py`, which uses the `DATADOG_CLIENT_TOKEN` and
+`DATADOG_APPLICATION_ID` environment variables to populate the necessary keys in the settings file.
+
+To run the application and send demo data to the environment, run `./run-demo.py`.
```

Comment:
I don't see such file. Am I missing something?

Reply:
Nope, not done yet. I was documenting eventual state rather than current state. I'll remove it for now.